frontend/src/app/modules/team/create-team/create-team.component.ts

```diff
@@ -20,13 +27,20 @@ export class CreateTeamComponent implements OnInit {
                     Validators.required,
                     Validators.minLength(3),
                     Validators.maxLength(50),
+                    Validators.pattern(regexs.teamName)
+                ],
+                [
+                    uniqueTeamNameValidator({} as Team, this.teamService)
                 ]
```

Comment:
Please, rewrite that validator as unique slag validator

Reply:
Fixed